src/events/http/HttpServer.js

```diff
@@ -326,7 +342,7 @@ export default class HttpServer {
         ? serverlessAuthorizerOptions?.payloadVersion || "2.0"
         : "1.0",
       resultTtlInSeconds:
-        serverlessAuthorizerOptions?.resultTtlInSeconds || "300",
+        serverlessAuthorizerOptions?.resultTtlInSeconds ?? "300",
```

Comment:
Originally I was copying more of the config from `serverlessAuthorizerOptions` but I un-did that to simplify the logic.  However, this needed to be changed in order to allow `0` to be set for `resultTtlInSeconds` inside the provider config.